web/hosts.go

```diff
@@ -52,9 +67,11 @@ func NewHostListHandler(client consul.Client) gin.HandlerFunc {
 		c.HTML(http.StatusOK, "hosts.html.tmpl", gin.H{
 			"Hosts":           hostList[firstElem:lastElem],
 			"SIDs":            getAllSIDs(hostList),
+			"Tags":            getAllTags(hostsTags),
```

Comment:
What's the difference between `Tags` and `HostsTags` here?

Reply:
Tags is the set of all the de-duplicated tags of all the hosts, and it is used to fill the options of the filter

Reply:
`HostsTags` is a map with the hostname as key and the tags of that host as value